distrobox-create

```diff
@@ -664,7 +675,7 @@ fi
 if ${container_manager} inspect --type container "${container_name}" > /dev/null 2>&1; then
 	printf "Distrobox named '%s' already exists.\n" "${container_name}"
 	printf "To enter, run:\n\n"
-	if [ "${rootful}" -ne 0 ]; then
+	if [ "${container_user_uid}" -ne 0 ] || [ "${rootful}" -eq 0 ]; then
```

Comment:
Put this way, seems like if `rootful=0` or `container_user_uid!=0` will always say to use `--root`

So If I use `distrobox create` as a normal user (no root, nor --root) I will receive this message, which is not correct

Reply:
You do the right check at 734

Reply:
This bit is a tricky one to get right. I'm gonna revise this carefully, though I remember this working as expected in my testings.

Reply:
You were right. I forgot to use the solution I've found, used at line 734, here.

Reply:
The documentation need changes as well, I almost forgot about them.